php_phongo.c

```diff
@@ -1599,20 +1593,35 @@ void phongo_manager_init(php_phongo_manager_t *manager, const char *uri_string,
 		}
 	}
 
+#ifdef MONGOC_ENABLE_SSL
+	/* Construct SSL options even if SSL is not enabled so that exceptions can
+	 * be thrown for unsupported driver options. */
 	ssl_opt = php_phongo_make_ssl_opt(driverOptions TSRMLS_CC);
 
 	/* An exception may be thrown during SSL option creation */
 	if (EG(exception)) {
 		goto cleanup;
 	}
+#else
+	if (mongoc_uri_get_ssl(uri)) {
+		phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT TSRMLS_CC, "Cannot create SSL client. SSL is not enabled in this build.");
```

Comment:
Although `mongoc_client_new_from_uri()` (invoked from `php_phongo_make_mongo_client()`) will log an error and return `NULL` if the SSL option is specified but SSL is not enabled in the build, it is better to catch this preemptively. The aforementioned error logging may or may not throw a specific exception depending on whether `mongodb.error` is enabled (see [PHPC-930](https://jira.mongodb.org/browse/PHPC-930)), but we can always throw a specific exception here.